### PR TITLE
Release 2023.02.2

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Release branch
- **Is this PR related to a Monolith release?** Yes. 2023.02.2

### Resolves:

* [OKTA-578706](https://oktainc.atlassian.net/browse/OKTA-578706)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md [hunk 1]
```diff
@@ -6,6 +6,22 @@ title: Okta Identity Engine API Products release notes 2023
...
+|------------------------------------------------------------------------------------------------------------------|-----------------|
+| [Bugs fixed in 2023.02.2](#bugs-fixed-in-2023-02-2)                                                              |February 29, 2023 |
+
+#### Bugs fixed in 2023.02.2
+
+* Some event hook requests failed to send in Preview orgs. (OKTA-578439)
+
+* Events weren't logged in the System Log when the Users API (`DELETE /users/${userId}/clients/${clientId}/tokens/${tokenId}`) was used to revoke refresh tokens. (OKTA-574992)
+
+* During an app's request to the `/authorize` endpoint, users who weren't assigned to the app could enroll an authenticator. (OKTA-575258)
```
Comment (isaacdurham-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:21]: Are possessives OK?

**Review by isaacdurham-okta — APPROVED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/27eb86d6b33603182a92466a28af39e37d5a8f89/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | 72 | 171 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2b444eb2-1e8e-411f-854e-9c804e0f2266) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/27eb86d6b33603182a92466a28af39e37d5a8f89/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | 75 | 114 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/68201c14-830d-4c7d-b104-e4d707b5dff7) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 2 of 2 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.338c58d8-8c85-4e9c-9ab2-3038507ee0ca?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md (hunk 1 above)
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:21]: @isaacdurham-okta The InfoDev style guide only prohibits using possessive with Okta (Okta's). According to MS style guide, they're fine: https://learn.microsoft.com/en-us/style-guide/punctuation/apostrophes

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/1c2617e5a91e1da3c3b2d894ba6cb0e075927080/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | 72 | 171 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c525924f-9041-4975-bdaa-49e262d524c3) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/1c2617e5a91e1da3c3b2d894ba6cb0e075927080/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | 76 | 112 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3a8a2d8a-8b2a-410b-929a-2704895d82e6) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 2 of 2 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.58599142-2028-4a87-9a5e-78b35c320ea1?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Pull request merged**